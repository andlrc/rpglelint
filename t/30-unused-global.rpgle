// rpglelint: -Wunused-variable

dcl-s global int(10);

dcl-proc proc;
  global = 1;
end-proc;
