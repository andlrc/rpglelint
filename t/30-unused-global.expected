t/30-unused-global.rpgle:3:7: warning: 'global' defined but not used [-Wunused-variable]
 dcl-s global int(10);
       ^
